sklearn/ensemble/tests/test_gradient_boosting_loss_functions.py

```diff
@@ -145,6 +149,59 @@ def test_sample_weight_deviance():
         assert deviance_wo_w == deviance_w_w
 
 
+@pytest.mark.parametrize('n_classes, n_samples',
+                         [(3, 100), (5, 57), (7, 13)])
+def test_multinomial_deviance(n_classes, n_samples):
+    # Check multinomial deviance with and without sample weights.
+    rng = check_random_state(13)
+    sample_weight = np.ones(n_samples)
+    y = rng.randint(0, n_classes, size=sample_weight.shape[0])
+    p = np.zeros((y.shape[0], n_classes), dtype=np.float64)
+    for i in range(p.shape[1]):
+        p[:, i] = y == i
+
+    loss = MultinomialDeviance(n_classes)
+    loss_wo_sw = loss(y, p)
+    assert loss_wo_sw > 0
+    loss_w_sw = loss(y, p, np.ones(p.shape[0], dtype=np.float32))
+    assert_almost_equal(loss_wo_sw, loss_w_sw)
+    loss_w_sw = loss(y, p, 0.5 * np.ones(p.shape[0], dtype=np.float32))
+    assert_almost_equal(loss_wo_sw, loss_w_sw)
+
+
+@pytest.mark.parametrize('pred, y, expected_loss',
+                         [(np.array([[1.0, 0, 0],
+                                     [0, 0.5, 0.5]]),
+                           np.array([0, 1]),
+                           0.75473)])
+def test_mdl_computation_unweighted(pred, y, expected_loss):
+    # MultinomialDeviance loss computation with uniform/without weights.
+    weights = np.array([1, 1])
+    loss = MultinomialDeviance(3)
+    assert_almost_equal(loss(y, pred, weights), expected_loss, decimal=4)
+    assert_almost_equal(loss(y, pred, None), expected_loss, decimal=4)
+    assert_almost_equal(loss(y, pred), expected_loss, decimal=4)
+
+
+@pytest.mark.parametrize('pred, y, weights, expected_loss',
+                         [(np.array([[1.0, 0, 0],
+                                     [0, 0.5, 0.5]]),
+                           np.array([0, 1]),
+                           np.array([1, 3]),
+                           0.85637)])
```

Comment:
```suggestion
@pytest.mark.parametrize(
    'pred, y, weights, expected_loss',
    [np.array([[1.0, 0, 0], [0, 0.5, 0.5]]), np.array([0, 1]),
     np.array([1, 3]), 0.85637]
)
```

Just a style change

Reply:
We don't a parametrization here. We can define the variable directly inside the test function.